assets/app/views/create/finished.html

```diff
@@ -0,0 +1,53 @@
+<div ng-controller="ProjectController">
+  <div ng-controller="CreateFinishedController" class="container">
+    <ol class="breadcrumb">
+      <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
+      <li><a href="project/{{projectName}}/create">Add to Project</a></li>
+      <li ng-if="name"><a href="#">{{name}}</a></li>
+      <li class="active"><strong>Next Steps</strong></li>
+    </ol>
+    <div class="row">
+      <div class="col-md-12">
+        <div class="create-finished">
+          <div class="row">
+            <div class="col-md-12">
+              <tasks></tasks>
+              <alerts alerts="alerts"></alerts>
+              
+              <h1>Next steps</h1>
+              <p>Go to the <a href="project/{{projectName}}/overview">overview page</a> to see more details.</p>
+
+              <h2 ng-if="createdBuildConfig">Making code changes</h2>
+              <div ng-repeat="trigger in createdBuildConfig.spec.triggers">
+                <div ng-switch="trigger.type">
+                  <div ng-switch-when="GitHub">
+                    <p>A GitHub <a target="_blank" href="https://docs.openshift.org/latest/dev_guide/builds.html#webhook-triggers">webhook trigger</a> has been created for the <strong>{{createdBuildConfig.metadata.name}}</strong> Build Config. You can now set up the webhook in your GitHub repository settings in <a target="_blank" href="{{createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks">{{createdBuildConfig.spec.source.git.uri | githubLink}}/settings/hooks</a>, using the following payload URL:</p>
```

Comment:
use the help topic directive so we can gather all help links... these need to change for OSE, for example